src_py/apiServer/experiment.py

```diff
@@ -9,20 +9,23 @@
 
 class Experiment():
 
-    def __init__(self, expName = "Untitled"):
+    def __init__(self ,expName = "Untitled"):
         self.name = expName
         self.trainingResList = []
         self.predictionResList = []
+    
+    def set_experiment_flow(self,expFlow):
+        self.expFlow = expFlow
 
     def syncTrainingWithFlow(self):
-       for source in globe.expFlow["Training"]:
+       for source in self.expFlow ["Training"]:
```

Comment:
Add the global var a definition (like in cpp): 
TRAINING_STR = "Training"
PREDICTION_STR = "Prediction" 
and use the variable. 

Think about the case that we want to change Prediciton to inference in the future (for example).

Reply:
Fixed